skynet/modules/ttt/processor.py

```diff
@@ -191,13 +118,14 @@ async def summarize(payload: DocumentPayload, job_type: JobType, model: BaseChat
     )
 
     # this is a rough estimate of the number of tokens in the input text, since llama models will have a different tokenization scheme
-    num_tokens = current_model.get_num_tokens(text)
+    num_tokens = model.get_num_tokens(text)
 
     # allow some buffer for the model to generate the output
+    # TODO: adjust this to the actual model's context window
```

Comment:
We will need to keep a mapping between models and context sizes. Right now we can make it simple since gpt4o is128k just like llama 3.1 so the only one with a slightly small window is the local 8b where we trim it.

We can do that in a follow-up PR, I just left this here so we don't forget :-)